manifests/init.pp

```diff
@@ -119,27 +119,39 @@
         }
         default:{
           case $version {
-            /^1\.8.*$/:{
+            /^1\.8(\..*)*$/:{
```

Comment:
That implies that there will always be a '.' after the minor version. That's not always been the case afaik?